core/container/externalbuilder/session.go

```diff
@@ -76,9 +84,13 @@ func (s *Session) waitForExit(logger *flogging.FabricLogger, stderr io.Reader) {
 // Wait waits for the running command to terminate and returns the exit error
 // from the command. If the command has already exited, the exit err will be
 // returned immediately.
-func (s *Session) Wait() error {
-	<-s.exited
-	return s.exitErr
+func (s *Session) Wait(optText ...string) error {
```

Comment:
What is `optText`? What is it used for?

Reply:
Oops, this was just put in by myself to debug my way through the flow and where exactly session.Wait was being called.